### ISSUE TITLE
File deletion timespan in trash should not roundet

### ISSUE THREAD

**Issue by RailsViceRoads** [nextcloud/server#24266]
**describtion**
The files app shows in the trash folder at the right side the timespan since this file was deleted. This timespan is translated to a human readable format like "a year ago". But in the tooltip I see (today) a day in 2020.
![grafik](https://user-images.githubusercontent.com/36618192/99854380-046f9b80-2b85-11eb-83aa-00bab41349a0.png)
Note: The screenshot was taken in 2020/11/20

**solution I would like**
Specially (or only?) for the trash I mean it is more intuitive, that the "was deleted time" is *at least* the timespan. If is displayed "a year ago" I sense the deletion is *at least* one year old. Not the deletion is *rounded* one year old. 

**notes**
I think this could prevent a to early permanent deletion of a file. 
May be this is also more intuitive for "modified at" timespans.

**Comment by szaimen** [comment]
cc @nextcloud/designers does this make sense?

**Comment by marcelklehr** [comment]
Not a designer, but I agree that rounding up timespans is confusing.

**Comment by nimisha-vijay** [comment]
Makes sense, maybe showing the exact date is unnecessary, but if using a rounded date make sure that it's rounded down (eg. if the exact date of deletion is 14 months ago, we can round down and say deleted 1 year ago, but if it's 10 months, then we can say something like 10 months without rounding it up to 1 year).
